### PR TITLE
docs: revise README file

### PR DESCRIPTION
**Pull request by tisonkun**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Read the diff.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,108 +23,102 @@ It's designed to work on infrastructure of the cloud era, and users benefit from
...
+- [GreptimeDB Erlang Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-erl)
+- [GreptimeDB Rust Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-rust)
+- [GreptimeDB JavaScript Ingester](https://github.com/GreptimeTeam/greptime-ingester-js)
 
 ### Grafana Dashboard
 
-Our official Grafana dashboard is available at [grafana](./grafana/README.md) directory.
+Our official Grafana dashboard is available at [grafana](grafana/README.md) directory.
 
 ## Project Status
```
Comment (tisonkun) [inline README.md:123]: We may need to update the project status also.

PTAL @killme2008 @sunng87 @fengjiachun

Comment (killme2008) [reply README.md:122]: I think its status is going to 1.0 GA.

**Review by killme2008 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (tisonkun) [reply README.md:122]: Well. Let me draft an edit in a follow-up.

This PR is large enough :D

**Pull request merged**